config/rate_limit.go

```diff
@@ -19,6 +19,10 @@ type RateLimit struct {
 	// The standard rate limiter offers similar performance as the sentinel-based limiter. This is disabled by default.
 	EnableSentinelRateLimiter bool `json:"enable_sentinel_rate_limiter"`
 
+	// EnableRateLimitSmoothing enables or disables rate smoothing. The rate smoothing is only supported on the
+	// Redis Rate Limiter, or the Sentinel Rate Limiter, as both algorithms implement a sliding log.
```

Comment:
Is there a reason why we can't implement this on DRL and on the new Fixed Window rate limiter?

Reply:
There's no reason we can't implement this on any rate algo that supports getting the current rate; DRL is not distributed and doesn't use redis, so it's a bit of a non-starter, since it's hard to reason about the global defined rate or even debug behaviour.